config/DeprecatedComponent.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from "react";
+
+const DeprecatedComponent = (Component: React$ComponentType<any>, text: string) => {
+  let warned = false;
+  return (props: Object) => {
```

Comment:
cool, just for future, it's possible to parametrize the types, I think something like:

```
const DeprecatedComponent = <Props>(Component: React$ComponentType<any>, text: string) => {
```

and then inside the function like `return (props: Props) => {`

Reply:
Ok, thanks 😊